03.image_classification/README.cn.md

```diff
@@ -474,39 +459,55 @@ Test with Pass 0, {'classification_error_evaluator': 0.885200023651123}
 
 ## 应用模型
 
-可以使用训练好的模型对图片进行分类，下面程序展示了如何使用`paddle.infer`接口进行推断，可以打开注释，更改加载的模型。
+可以使用训练好的模型对图片进行分类，下面程序展示了如何使用 `fluid.Inferencer` 接口进行推断，可以打开注释，更改加载的模型。
+
+### 生成预测输入数据
+
+`dog.png` is an example image of a dog. Turn it into an numpy array to match the data feeder format.
 
 ```python
+# Prepare testing data.
 from PIL import Image
 import numpy as np
 import os
+
 def load_image(file):
     im = Image.open(file)
     im = im.resize((32, 32), Image.ANTIALIAS)
+
     im = np.array(im).astype(np.float32)
-    # PIL打开图片存储顺序为H(高度)，W(宽度)，C(通道)。
-    # PaddlePaddle要求数据顺序为CHW，所以需要转换顺序。
-    im = im.transpose((2, 0, 1)) # CHW
-    # CIFAR训练图片通道顺序为B(蓝),G(绿),R(红),
-    # 而PIL打开图片默认通道顺序为RGB,因为需要交换通道。
-    im = im[(2, 1, 0),:,:] # BGR
-    im = im.flatten()
+    # The storage order of the loaded image is W(widht),
+    # H(height), C(channel). PaddlePaddle requires
+    # the CHW order, so transpose them.
+    im = im.transpose((2, 0, 1))  # CHW
+    # In the training phase, the channel order of CIFAR
+    # image is B(Blue), G(green), R(Red). But PIL open
+    # image in RGB mode. It must swap the channel order.
+    im = im[(2, 1, 0), :, :]  # BGR
     im = im / 255.0
+
+    # Add one dimension to mimic the list format.
+    im = numpy.expand_dims(im, axis=0)
     return im
 
-test_data = []
 cur_dir = os.getcwd()
-test_data.append((load_image(cur_dir + '/image/dog.png'),))
+img = load_image(cur_dir + '/image/dog.png')
+```
 
-# with open('params_pass_50.tar', 'r') as f:
-#    parameters = paddle.parameters.Parameters.from_tar(f)
+### Inferencer 配置和预测
 
-probs = paddle.infer(
-    output_layer=out, parameters=parameters, input=test_data)
-lab = np.argsort(-probs) # probs and lab are the results of one batch data
-print "Label of image/dog.png is: %d" % lab[0][0]
-```
+`Inferencer` 需要一个 `infer_func` 和 `param_path` 来设置网络和经过训练的参数。
```

Comment:
kind of weird that we call the param name `param_path` but use `params_dirname` as the variable name 😭 

`param_path=params_dirname`

But let's keep it as-is for now.